docs/source/tutorials/plots_intro.ipynb

```diff
@@ -530,6 +530,7 @@
    "id": "5211f6de-d447-46a8-8652-3c9fd77d7915",
```

Comment:
<p>Nit: Start the sentence with "for example" -&gt; "For example, we can use it in <code>plot_dist</code> to control..." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz-plots/pull/261/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>